vega_sim/api/data.py

```diff
@@ -363,6 +389,9 @@ def _order_from_proto(
         updated_at=order.updated_at,
         version=order.version,
         market_id=order.market_id,
+        iceberg_order=_iceberg_order_from_proto(order.iceberg_order, decimal_spec)
+        if order.HasField("iceberg_order")
+        else None,
     )
```

Comment:
At some point these could be made into `classmethod`s on the data classes themselves, though probably not now